src/PIL/Image.py

```diff
@@ -508,22 +509,22 @@ def __init__(self):
         self._exif = None
 
     @property
-    def width(self):
+    def width(self) -> int:
         return self.size[0]
 
     @property
-    def height(self):
+    def height(self) -> int:
         return self.size[1]
 
     @property
-    def size(self):
+    def size(self) -> tuple[int, int]:
         return self._size
 
     @property
     def mode(self):
         return self._mode
 
-    def _new(self, im):
+    def _new(self, im: Image):
```

Comment:
```suggestion
    def _new(self, im) -> Image:
```
This was corrected the wrong way around - it returns an `Image`, but `im` is not.